### DISCUSSION TITLE
New Update works a bit different

### DISCUSSION THREAD

**Discussion by scarecrow2k6** [yandeu/five-server#8] [Help / Questions] [answered]
First, thank you for five-server, it is indeed a great extension. However, the latest update is doing things a bit differently, don't know if it is intentional or things are just different on my end. I am using the latest version of VSCode.

With update v0.0.28, when I click on "Go Live" or "Open with Live Five Server" on VSCode. The browser won't pop up with the current active page like before. I would have to go down and click on the links from the Terminal. It is not a hassle but just wondering if it was intentional.

Another thing is that after opening the link the page won't be the current active html file but some other tab. Clicking on another HTML file tab in VSCode would update to that page. So to view the intended HTML file, I would have to click on that tab to view it.

**Comment by yandeu** [comment, marked as answer]
> browser won't pop up

You're right. This is not intentional. I haven't noticed it until now. I will fix it in the next version.

> won't be the current active html file

Do you mean clicking the link in the terminal? Yes, this will open the root and not the current file.
*Once the above issue is resolved, you won't need to click the link in the terminal anymore.*

---

**EDIT:**
I found the issue and fixed it in https://github.com/yandeu/five-server/commit/92625a9a243f131f14eebdcad3cd2108b12eefad.

Explanation: In Five Server v0.0.28 (VSCode), the option `browser` is an empty array by default, which is invalid.
